### PR TITLE
Design/single

### PR DESCRIPTION
**Pull request by taisuou**
single page template
- page menu sidebar<aside>
- markdown body<div>
-current menu sidebar<aside>

js development has not finished yet.
but Tags have classes and styles to handle with js

ex.)
page menu sidebar : open and close, show arrow before <ul>
current menu : open and close, change bar color, change position fixed when schrolled.

### CODE REVIEW

File: themes/vald/layouts/_default/single.html [hunk 1]
```diff
@@ -1,5 +1,70 @@
...
+
+
+
+
+
+　　
+　　
+
+　　
+
```
Comment (vankichi) [inline themes/vald/layouts/_default/single.html:70]:
```suggestion
```

File: themes/vald/static/css/main.css [hunk 2]
```diff
@@ -266,44 +269,416 @@
...
   }
+
   .mdl-card__caption {
     height: 72px;
   }
 }
+
+
+
+
```
Comment (vankichi) [inline themes/vald/static/css/main.css:327]:
```suggestion
```

Comment (lodgefab) [reply themes/vald/static/css/main.css:327]: thank you for deleting blank line

Comment (taisuou) [reply themes/vald/static/css/main.css:327]: スクリーンショット 2020-06-01 10 39

**Pull request merged**